yt_dlp/extractor/youku.py

```diff
@@ -87,6 +87,14 @@ class YoukuIE(InfoExtractor):
             'uploader_url': 'https://www.youku.com/profile/index/?uid=UNjU2MzY1MzM1Ng==',
             'tags': list,
         },
+    }, {
+        'url': 'https://play.tudou.com/v_show/id_XNjAxNjI2OTU3Ng.html?',
+        'info_dict': {
+            'id': 'XNjAxNjI2OTU3Ng==',
+            'ext': 'mp4',
+            'title': '阿斯塔意识到哈里杀了人，自己被骗了',
+            'show_name': '外星居民 第一季',
```

Comment:
```suggestion
            'id': 'XNjAxNjI2OTU3Ng',
            'ext': 'mp4',
```
Make the above change, and then locally run this command from the project root directory:
`python test/test_download.py TestDownload.test_Youku_4`
it will fail and output the rest of the info_dict you need to add to the test. Add it and commit

Reply:
> Make the above change, and then locally run this command from the project root directory: `python test/test_download.py TestDownload.test_Youku_4` it will fail and output the rest of the info_dict you need to add to the test. Add it and commit

Yeah I just realised info returned from youku.py is a bit different, checking...